### ISSUE TITLE
isce3 based processing wrapped in PyroSAR

### ISSUE THREAD

**Issue by whatnick** [johntruckenbrodt/pyroSAR#322]
It would be interesting to wrap [isce3](https://github.com/isce-framework/isce3) + python API in the pyrosar processing pipeline since it has efficient C++ codebase and is opensource.

**Comment by johntruckenbrodt** [comment]
Absolutely! I've been playing around a bit with ISCE3 but so far have not had the time to do any further integration.  
I wouldn't say this is a bug though :wink:.

**Comment by johntruckenbrodt** [comment]
Thanks for picking this up again! Before we get started, let's consider the use cases. Are you interested in a general Python interface for ISCE or "just" a Python wrapper for a specific use case? For example, there is this one to process RTC products:  
https://github.com/opera-adt/RTC

This is mostly Python and surely a good starting point.

I am on leave in June so can't actively work on it but we can start talking it through.

**Comment by whatnick** [comment]
Sounds like a great starting point as a facade. This is going to be the
most likely usage pattern.

**Comment by johntruckenbrodt** [comment]
Might be of interest: https://github.com/forrestfwilliams/MultiRTC
